### DISCUSSION TITLE
Option to keep files selected after move or copy

### DISCUSSION THREAD

**Discussion by advice2020** [doublecmd/doublecmd#1378] [Suggestions]
When you use for example F5 to "copy to" multiple selected files from focused file pane to destination file pane, these files deselect automatically after the action occurs.
Also after files are copied or moved into the destination file pane they are also not selected.

Maybe can add two options one for FOCUSED file pane & one for DESTINATION file pane.
1.) Keep files in FOCUSED file pane selected after copied items to DESTINATION file pane
2.) Keep files in the DESTINATION file pane selected after files have been moved to / copied to

Thank You

**Comment by ags1234** [comment]
This would be great! I am also interested in seeing it implemented.
